src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs

```diff
@@ -1239,7 +1239,7 @@ public void LangVersion()
             parsedArgs = DefaultParse(new[] { "/langversion:latest", "a.cs" }, _baseDirectory);
             parsedArgs.Errors.Verify();
             Assert.Equal(LanguageVersion.Latest, parsedArgs.ParseOptions.SpecifiedLanguageVersion);
-            Assert.Equal(defaultVersion, parsedArgs.ParseOptions.LanguageVersion);
+            Assert.Equal(LanguageVersion.CSharp7_1, parsedArgs.ParseOptions.LanguageVersion);
```

Comment:
A test like that can be completely based on LanguageVersion members and doesn't really belong in CommandLineTests because we want specific value regardless of the context.